embed-api-v3.rst

```diff
@@ -0,0 +1,119 @@
+.. post:: December 2, 2021
+   :tags: api, feature, grant
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      We are thrilled to announce the availability of Read the Docs Embed API v3,
+      along with its official client, sphinx-hoverxref 1.0.
+      We invite the community to try them out and let us know their feedback.
+
+Announcing Embed API v3 and sphinx-hoverxref 1.0
+================================================
+
+We are thrilled to announce the availability of Read the Docs Embed API v3,
+along with its official client, sphinx-hoverxref 1.0.
+This work has been possible in part thanks to the
+:doc:`the CZI grant we received </czi-grant-announcement>`.
+
+Why using content embedding?
+----------------------------
+
+As we wrote in
+:doc:`our first blog post about sphinx-hoverxref </hoverxref-intersphinx>`,
+one of the most powerful features of Sphinx
+is the possibility of creating cross references
+to other documentation projects.
+However, a reader finding several links in a technical documentation
+might need to open several browser tabs to fully understand the context,
+resulting in a lot of friction in the form of context switching.
+
+One way of mitigating the problem is to show a small preview
+in the form of tooltip or modal with the contents of the link.
+This is where the Embed API and sphinx-hoverxref come in.
+
+`sphinx-hoverxref`_ is a Sphinx extension that addresses this problem
+by adding tooltips on the cross references of the HTML documentation
+with the content of the linked section.
+To do so, it inserts a small JavaScript file that
+calls the Read the Docs Embed API when the user hovers any link.
+:doc:`Back in June </hoverxref-intersphinx>` we announced
+intersphinx support in `sphinx-hoverxref`_ 0.6b1,
+which allowed users to include modals and tooltips
+also for cross references to other projects.
```

Comment:
Does 1.0 have any new features? Seems weird to talk about old features that we already wrote about. I'd probably highlight the main valuable features of the project itself, instead of a subset of the last released version.